modules/ocf_mirrors/manifests/trisquel.pp

```diff
@@ -11,4 +11,11 @@
       rsync_extra              => '--block-size=8192',
       cron_minute              => '55';
   }
+
+  ocf_mirrors::monitoring { 'trisquel':
+    type          => 'debian',
+    dist_to_check => 'flidas-security',
```

Comment:
Why the security dist? Just curious, I'd imagine the main dist (`flidas`) or the updates dist (`flidas-updates`) would get updated more often, but I haven't done much research here.

Reply:
This is troublesome because `flidas` won't be updated after 2021, after which this healthcheck will silently always succeed. On the other hand, files like `bandwidth-day.png` seem to get updated every day...

Reply:
I mean, yeah, that date should be in the headers, but I'd really prefer to wait until 2021 to give trisquel a unique healthcheck...

Reply:
However, by 2021 everyone will have forgotten that this needs to be updated.

Reply:
Honestly I think this is fine as it is, we'll probably have changed our mirroring by then anyway, and if we haven't, then it doesn't hurt to have this over no monitoring at all. I do agree we should try and make this more dynamic somehow, but not sure how to do it.

Reply:
I mean, isn't trisquel 8 (`fildas`) still in alpha? We can update this when newer versions are released. I don't think it's a big deal, nor will it silently fail as new releases are publicly announced, and we will have to change one line in puppet.

Reply:
My concern is that people will forget to update it, and then it will silently fail.